satpy/etc/readers/seviri_l2_grib.yaml

```diff
@@ -276,41 +302,46 @@ datasets:
 
   nir_refl_39:
     name: nir_refl_39
+    long_name: TOA Bidirectional Reflectance at 3.9um
+    standard_name: toa_bidirectional_reflectance
     resolution: 3000.403165817
     wavelength: [3.48, 3.92, 4.36]
     file_type: grib_seviri_crm
     parameter_number: 12
     units: "%"
-    long_name: nir_refl_39
 
   num_accumulations:
     name: num_accumulations
+    long_name: CRM Product Number of Accumulations
     resolution: 3000.403165817
     file_type: grib_seviri_crm
     parameter_number: 6
     units: "1"
-    long_name:  num_accumulations
+    standard_name: number_of_accumulations
```

Comment:
can you please move this up under the `long_name` so that the order is same for all the datasets in this file.